.github/workflows/release.yaml

```diff
@@ -27,5 +27,7 @@ jobs:
           node-version-file: './package.json'
       - run: pnpm install
       - uses: changesets/action@v1
+        with:
+          publish: pnpm changeset:publish
```

Comment:
This property is needed to create Github release and npm package through changesets/action